menu_display.c

```diff
@@ -0,0 +1,113 @@
+#include <stdio.h>
+#include <stdarg.h>
+#include <stdlib.h>
+#include <windows.h>
+#include <windef.h>
+#include <winnt.h>
+#include <winbase.h>
+#include <string.h>
+#include <math.h>
+#include <stdbool.h>
+#include <stdint.h>

```

Comment:
Please just include the needed library header files. Some of them are not used, please remove them.